draft-ietf-rats-endorsements.md

```diff
@@ -201,13 +201,33 @@ some minimal matching policy (e.g., exact match against a singleton reference
 value).  This unfortunately complicates design as a Verifier may need
 multiple parsers for matching policies.
 
-# Endorsing Identity
+# Key Material for Attestation
```

Comment:
Sorry, I missed the change of title.
I think this "endorsing identity" is a valid and useful conceptualisation.
What we are talking about here is precisely the endorsements that allow a verifier to know that a certain cryptographic key "speaks for" a known and genuine attester, i.e., the mechanism through which its identity is anchored.

Reply:
I think the changed title is better since "Endorsing Identity" doesn't seem right when DAA is used but the new title and the text in the section still applies in that case.

Reply:
IIUC, the DAA Issuer endorses a _group_ identity, but an identity still.

Reply:
The rest of the WG pushed back against the concept of "identity" per se being the focus.  Is there a technical objection to the change in section title?

Reply:
If you are referring to the conversation starting at https://mailarchive.ietf.org/arch/msg/rats/hEnwNxvLHcLV7CQGmwQvYwztGqI/ 
there was some initial pushback from Henk and Laurence (Ned's points were more neutral), to which I replied to clarify my point further.
The conversation stopped there. So, from my perspective, it's still in limbo.

@laurencelundblade, @henkbirkholz can you please look at that thread again and see if my reply clarifies your initial confusion?

Reply:
I checked. And then got caught up in other things. Continuing...

Appraisal of Evidence is always cut into two pieces:
1.) appraisal of Evidence provenance (most prominently signature checking in object security and peer authentication in secure channels) that identifies an Attesting Environment.
2.) appraisal of Evidence Claims that identifies or classifies a Target Environment AND are ultimately used to assess the trustworthiness that Target Environment.

So, the Attesting Environment is either endorsed or it was a Target Environment before and it was assessed to be trustworthy and now takes on the duty of an Attesting Environment. Following that line of though, on the one hand "Endorsed Identity" intuitively says "Attesting Environment" to me.

"Key Material for Attestation" could be anything, right? Text above talks about the "trust anchor actual state scenario" where the Actual State would be a trust anchor used to verify Evidence or Endorsement. The text here is talking about Evidence and somewhat Layered Attestation now. So I do not think it is about Endorsement authenticity, but only Evidence authenticity.

Could "Key Material for Attestation" simply be renamed to "Key Material for Evidence Provision and Appraisal"? Or should we massage the text to clearly include Evidence and Endorsement Authenticity?

Reply:
> [...] Following that line of though, on the one hand "Endorsed Identity" intuitively says "Attesting Environment" to me.

Yes, exactly.  The section is all about endorsing the authenticity of the (bottom-most) attesting environment.
And, BTW, I am fine with it.

> "Key Material for Attestation" could be anything, right?

Yes, my main concern with the proposed new title is it's too broad.  We are talking about a pretty specific use of crypto, i.e., how the verifier uses it to assess evidence's provenance.
The relevant thing is not crypto key material per se, but how we tell the verifier what key is the one that verifies the authenticity of AE's evidence statement.

> Could "Key Material for Attestation" simply be renamed to "Key Material for Evidence Provision and Appraisal"?

I don't think that makes the point any clearer.
I like your breakdown above (provenance vs claims), and I think.a more precise title would be "Endorsing Evidence Provenance".

> Or should we massage the text to clearly include Evidence and Endorsement Authenticity?

No, please.  This section has a nice and narrow scope.  Let's not bloat it.

Reply:
Ok, changed to "Endorsing Evidence Provenance"